### PR TITLE
Add wrapper to right-controls

### PR DESCRIPTION
**Pull request by corradio**
I've added a wrapper to start wrapper controls that appear on the right.
For now I've only wrapped the toggle but I'd be good if we could put them all there!

### CODE REVIEW

File: web/src/layout/main.js [hunk 1]
```diff
@@ -57,16 +57,17 @@ const Main = ({ brightModeEnabled, electricityMixMode, isLeftPanelCollapsed }) =
             </a>
           </div>
           <Legend />
-          <Toggle
-            className="production-consumption-toggle"
```
Comment (fbarl) [inline web/src/layout/main.js:61]: Good idea with the wrappers! I think it's ok to remove the `className` prop from the`Toggle` component if we're going to be using wrappers instead.

**Comment by corradio** [comment]
@fbarl I've added a commit that makes sure that a click on "consumption" switches to "consumption" instead of cycling through them.
This is necessary when we have more than two items
image

**Comment by fbarl** [comment]
> @fbarl I've added a commit that makes sure that a click on "consumption" switches to "consumption" instead of cycling through them. This is necessary when we have more than two items

Good call, it makes more sense that way! :+1:

File: web/src/components/toggle.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ const InfoButton = styled.div`
   }
 `;
 
-const Toggle = ({
+export default ({
   className,
```
Comment (fbarl) [inline web/src/components/toggle.js:77]: Not sure if I made myself clear enough, but in https://github.com/tmrowco/electricitymap-contrib/pull/2350#discussion_r402182450, I was suggesting removing this prop if it's not being used anymore.

File: web/src/components/toggle.js [hunk 3]
```diff
@@ -82,15 +82,11 @@ const Toggle = ({
 }) => {
   const [tooltipVisible, setTooltipVisible] = useState(false);
 
-  const activeIndex = findIndex(options, { value });
```
Comment (fbarl) [inline web/src/components/toggle.js:85]: I guess `findIndex` is not used here anymore, so it doesn't have to be imported (which reminds me, we should probably add a linting rule for that :wink:)

File: web/src/components/toggle.js (hunk 2 above)
Comment (corradio) [reply web/src/components/toggle.js:77]: ah got it! Will do

**Pull request merged**